### PR TITLE
QPR.7

### PR DESCRIPTION
**Pull request by harinineelu**

### CODE REVIEW

**Comment by kate-harrison** [comment]
Can you please take a look at this PR? Thanks!

**Comment by janetliuboston** [comment]
Yes, will take a look.

Janet

On Apr 16, 2018, at 6:43 PM, Kate <notifications@github.com<mailto:notifications@github.com>> wrote:


Can you please take a look at this PR? Thanks!

File: src/harness/testcases/WINNF_FT_S_QPR_testcase.py [hunk 1]
```diff
@@ -176,3 +185,127 @@ def test_WINNF_FT_S_QPR_6(self, config_filename):
...
+    # Register CBSD.
+    request = {'registrationRequest': config['registrationRequest']}
+    response = self._sas.Registration(request)['registrationResponse']
+
+    # Check registration response.
+    self.assertEqual(len(response), len(config['registrationRequest']))
+    if response[0]['response']['responseCode'] != 0:
+      return  # SAS passes immediately in this case.
+    cbsd_id = response[0]['cbsdId']
+
```
Comment (janetliuboston) [inline src/harness/testcases/WINNF_FT_S_QPR_testcase.py]: Suggest:
del request, response

**Comment by harinineelu** [comment]
Thanks Janet, Added the suggested line. PTAL.

File: src/harness/testcases/WINNF_FT_S_QPR_testcase.py (hunk 1 above)
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_QPR_testcase.py]: Done

**Comment by harinineelu** [comment]
Merged #522

**Pull request merged**